ansible/roles/ansible-datamover-api/tasks/create_dmapi_db.yml

```diff
@@ -0,0 +1,26 @@
+- import_tasks: db_setup.yml
+  when:
+    - inventory_hostname == groups['dmapi_all'][0]
```

Comment:
use run_once: true instead of the conditional check

Reply:
changes done